website/docs/docs/dbt-cloud-apis/sl-python-sdk.md

```diff
@@ -101,6 +101,78 @@ asyncio.run(main())
 
 ```
 
+### Lazy loading for large fields
+
+By default, the Python SDK eagerly loads nested lists of objects such as `dimensions`, `entities`, and `measures` for each `Metric` &mdash; even if you don't need them. This is generally convenient, but in large projects, it can lead to slower responses due to the amount of data returned.
+
+To improve performance, you can opt into lazy loading by passing `lazy=True` when creating the client. With lazy loading enabled, the SDK skips fetching large nested fields until you explicitly request them on a per-model-basis. 
+
+Lazy loading is currently only supported for `dimensions`, `entities`, and `measures` on `Metric` objects.
```

Comment:
At the top of this we say "such as" and here we say "only." Might be helpful to combine the two and just call out at the top that these are the only types supported